tests/acceptance/default_ncf.cf.sub

```diff
@@ -125,39 +128,55 @@ bundle agent apply_gm(name, args, status, result_class)
       "8_arg" expression => strcmp("${length}", "8");
 
     pass2::
-      "old_class_prefix_ok" expression => "${define_expected_classes.report_string_old_class_prefix}";
-      "class_prefix_ok"     expression => "${define_expected_classes.report_string_class_prefix}";
+      "old_class_prefix_ok" expression => "${define_expected_classes.report_string_old_class_prefix_${result_class}}";
+      "class_prefix_ok"     expression => "${define_expected_classes.report_string_class_prefix_${result_class}}";
       "${result_class}_ok"  expression => "old_class_prefix_ok.class_prefix_ok",
         scope => "namespace";
 
 
   methods:
     pass1::
```

Comment:
why pass1 and not leaving the class condition empty ?

Reply:
No reasons, I just wrote it this way when trying to figure out a bug which could has come from the pass.
I will fixup this